### DISCUSSION TITLE
Export to CSV does not do anything

### DISCUSSION THREAD

**Discussion by luifire** [datalust/seq-tickets#1790] [Help]
Hi SEQ Team

I have a simple query, which I want to export to CSV.

```
select (max(A.B.C)) as C
from stream 
group by SomeId
order by C
limit 100
```

When I press "Export as CSV" (the "," Symbol).
It says "Export started", but nothing happens.
The query itself executes as expected. I tried different browsers, but no difference.
I searched my entire disk, but there is no new CSV file.
Judging from the task manager, it uses the disk for a few seconds, everytime I press "Export as CSV".

![image](https://user-images.githubusercontent.com/53562633/223969698-ba3786fd-789e-4e89-a5bc-45c3435dd4b3.png)

**Comment by liammclennan** [comment]
Hi @luifire 

Unfortunately when I tried to export your query results to CSV in Firefox and Chrome I could not reproduce a problem.

Please send to support@datalust.co:

* your version of Seq
* the browsers you tried
* any errors in the browser developer tools console
* a fresh diagnostics report from Settings > Diagnostics

**Comment by liammclennan** [comment]
A fix has been merged and will be in the next Seq release.

**Reply by luifire** [reply]
Thank you :)
